app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
+<p><%= link_to 'Confirm my account', user_confirmation_url(confirmation_token: @token) %></p>
```

Comment:
- [ ] Greate job on implementing all the requirements, but I can't get the confirmation token to verify the inputs.

![Capture web_1-3-2022_122837_localhost](https://user-images.githubusercontent.com/61562626/156161353-bd50295b-7166-4c31-a2bb-e2259ffddde6.jpeg)

Reply:
@dasileker 
hi Amine ,
As I have showed you in our meeting , the app works smoothly in my system without any issue. So I will just ask for an other review as you had suggested me.

Reply:
@Taher-web-dev It's weird that the app is functioning in your environment. I hope this problem is on my side.